src/pydata_sphinx_theme/theme/pydata_sphinx_theme/layout.html

```diff
@@ -17,6 +17,15 @@
     document.documentElement.dataset.mode = localStorage.getItem("mode") || "{{ default_mode }}";
     document.documentElement.dataset.theme = localStorage.getItem("theme") || "{{ default_mode }}";
   </script>
+  <!-- 
+    this give us a css class that will be invisible only if js is disabled 
+  -->
+  <noscript>
+    <style>
+      *.jsonly, *.btn.jsonly { display: none; } 
```

Comment:
+1 to pst-, but I woulld prefer not to use `!important`, I have been told it is bad practice.

Reply:
So that goes back to my original question: is the leading asterisk for precedence? 

My concern is that it might be too easy right now for some other CSS with `display: flex` or some other display rule to override the `pst-js-only` class.

I'll need to take a closer look at selector precedence and come back to this.

Reply:
So yes, it's for precedence.

But I think we use pst-js-only in so few places – and I believe it should be used sparingly – that it should be ok to review each of them independently.

Alternative is to not have a class but a lit of ids of elements to hide when no js.

Reply:
One thing to note about `!important` is that it reverses the order of specificity: important user-agent rules come before important user rules come before important site author rules, which means that if a user or browser wants to override our pst-js-only rule, they can.